cf_remote/aramid.py

```diff
@@ -28,6 +28,7 @@
 import subprocess
 import time
 from urllib.parse import urlparse
+from cf_remote import log
```

Comment:
Before this change, `aramid` was an independent thing which didn't rely on anything from `cf_remote`. Now you've introduced a dependency on something from `cf_remote`, which means `aramid` can no longer be imported on its own as a separate module / library.

I think it's okay, but we could reverse this by:

Switching to only using python standard logging (https://docs.python.org/3/library/logging.html)

Or

Sending in a `log` object as a parameter when using aramid functions.